templates/zerver/api/changelog.md

```diff
@@ -10,6 +10,12 @@ below features are supported.
 
 ## Changes in Zulip 4.0
 
+**Feature level 60**
+
+* [`POST /register`](/api/register-queue): Added a new boolean field
+  `is_moderator`, similar to the existing `is_admin`, `is_owner` and
+  `is_guest` fields, to the response.
```

Comment:
I'm OK with including this sort of helper detail here, for API clients, since it's like 100x more useful to know whether you're a `moderator` than whether another user is for writing client code; these are very wide data structures anyway.